aten/src/ATen/cpu/vec/vec256/vec256_bfloat16.h

```diff
@@ -431,6 +431,15 @@ template <> class Vectorized<BFloat16> {
     auto o2 = _mm256_xor_ps(mask, hi);
     return cvtfp32_bf16(o1, o2);
   }
+  Vectorized<BFloat16> nextafter(const Vectorized<BFloat16> &b) const {
```

Comment:
Btw, I am curious to see if it works.

(I had tried casting to float, doing `nextafter` and casting back to bfloat16 while adding non-vectorized support). 
That did not work as the value of `nextafter` in float is much smaller than the precision of `bfloat16` and ends up being discarded when converted back to `bfloat16`.
Or
Maybe I did something incorrectly

Reply:
There was no difference in the precision of bfloat16 between non-vectorized supporting and vectorized supporting.